legacy/application-abi.md

```diff
@@ -51,6 +51,13 @@ Regardless of the kind, all modules accessing WASI APIs also export a table
 with the name `__indirect_function_table`. Function pointers in WASI API calls
 are relative to this table's index space.
 
+When `_start` or `_initialize` is called, environments shall provide file
+descriptors with indices 0, 1, and 2 representing file-like resources for
```

Comment:
what do you mean by "file-like"?
the wording made me think regular files. but i guess it isn't what you meant here.

Reply:
That's a good point, "file-like" is ambiguous. I've now changed it to say "stream" instead.